gatsby-config.js

```diff
@@ -94,6 +97,17 @@ module.exports = {
                 url: "https://etclabscore.github.io/nodes-interface/"
               }
             ]
+          },
+          {
+            heading: "Partners",
+            lastColumnItem: true,
+            links: [
+              {
+                title: "Rivet Blockchain Gateway",
+                image: "https://rivet.cloud/img/rivet.3ae58e03.svg",
```

Comment:
@bobsummerwill can you ask Rivet to provide us with a remote URL for their latest asset for branding?

Reply:
Resolved by 74e7f83